CMakeLists.txt

```diff
@@ -1,130 +1,76 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5.1)
 
-OPTION(USE_HUNTER "Use Hunter cmake package handler" OFF)
+cmake_policy(VERSION 3.5)
 
-if(USE_HUNTER)
-    include(${CMAKE_CURRENT_LIST_DIR}/CMakeModules/HunterGate.cmake)
-    
-    HunterGate(
-        URL "https://github.com/ruslo/hunter/archive/v0.18.43.tar.gz"
-        SHA1 "d2c8c42cd07f7cefe18fd9a9b9c13114b1a15a27"
-#        LOCAL
-    )
-endif()
+project(Forge VERSION 1.1.0 LANGUAGES C CXX)
+
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/CMakeModules")
+set(CMAKE_PREFIX_PATH "${CMAKE_BINARY_DIR}prefix;${CMAKE_PREFIX_PATH}")
 
-PROJECT(FORGE)
+include(BuildType)
+include(CMakeDependentOption)
+include(ConditionalDirectory)
+include(DependencyCheck)
+include(Version)
+include(FGInstallDirs)
 
-SET_PROPERTY(GLOBAL PROPERTY USE_FOLDERS ON)
-SET(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/CMakeModules")
-INCLUDE(FGInstallDirs)
-INCLUDE(Version)
+find_package(Doxygen QUIET)
+find_package(glbinding REQUIRED)
+find_package(glm REQUIRED)
+find_package(FreeType)
 
+option(BUILD_DOCS      "Build Documentation"               ${DOXYGEN_FOUND})
+option(BUILD_EXAMPLES  "Build Examples"                    ON)
+option(WITH_FREEIMAGE  "Use FreeImage to save framebuffer" ON)
 
-############################## BEGIN - CMAKE OPTIONS ##########################
-OPTION(BUILD_DOCUMENTATION "Build Documentation" OFF)
-OPTION(BUILD_EXAMPLES "Build Examples" ON)
-OPTION(USE_FREEIMAGE "Use freeimage to allow saving of charts" ON)
+find_package(Boost)
+
+if(Boost_FOUND AND NOT TARGET Boost::boost)
+    add_library(Boost::boost INTERFACE IMPORTED)
+    set_property(TARGET Boost::boost PROPERTY INTERFACE_INCLUDE_DIRECTORIES ${Boost_INCLUDE_DIRS})
+endif(Boost_FOUND AND NOT TARGET Boost::boost)
+
+if(UNIX)
```

Comment:
This is an unnecessary if condition. The packages will just not be found on Windows.